extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -13,48 +13,51 @@ public static function onParserFirstCallInit( Parser $parser ) {
 		global $wgEnableDiscussions;
 
 		if ( $wgEnableDiscussions ) {
-			$parser->setHook( self::TAG_NAME, [ 'EmbeddableDiscussionsController', 'render' ] );
+			$parser->setHook( static::TAG_NAME, [ 'EmbeddableDiscussionsController', 'render' ] );
```

Comment:
Is there a reason to make this static?  Typically this would be used if this was a parent class function being called from a child, e.g.:
```php
class Parent {
    const TYPE = 'default'
    public static function foo() {
       print "Type: " . static::TYPE . "\n";  // Use TYPE from the class being invoked
    }
class Child extends Parent {
    const TYPE = 'something-specific';
}

Child::foo(); // Prints 'something-specific'
Parent::foo(); // Prints 'default'
```